sensing/autoware_pointcloud_preprocessor/schema/concatenate_pointclouds.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Concatenate Pointclouds Node",
+  "type": "object",
+  "definitions": {
+    "concatenate_pointclouds": {
+      "type": "object",
+      "properties": {
+        "timeout_sec": {
+          "type": "number",
+          "default": 0.1,
```

Comment:
Assign default values as string

Reply:
Thanks! fixed them in https://github.com/autowarefoundation/autoware.universe/pull/8509/commits/08ec2189e062a6b8e4cb21748c703469a4120e0c